02week/ticTacToe.js

```diff
@@ -25,14 +25,27 @@ function printBoard() {
 
 function horizontalWin() {
   // Your code here
+  if (array1===[x,x,x]||[o,o,o]){
```

Comment:
Creative approach but you could check against the `playerTurn` variable here instead,

This will avoid checking for 'o' wins when `playerTurn === x` and vice versa,